tests/Cluster/ClusterInfoTest.cpp

```diff
@@ -120,3 +121,53 @@ TEST_F(ClusterInfoTest, testServerAliasExists) {
   }
 }
 
+TEST_F(ClusterInfoTest, plan_will_provide_latest_id) {
+  auto &cache {server.getFeature<ClusterFeature>().agencyCache()};
+  auto [acb, index] {cache.read({AgencyCommHelper::path("Sync/LatestID")})};
+  auto expectedLatestId {acb->slice().at(0).get("arango").get("Sync").get("LatestID").getInt()};
+  auto& ci {server.getFeature<arangodb::ClusterFeature>().clusterInfo()};
+  auto builder {std::make_shared<VPackBuilder>()};
+  auto result {ci.agencyPlan(builder)};
+  ASSERT_TRUE(result.ok());
+  ASSERT_EQ(builder->slice().at(0).get("arango").get("Sync").get("LatestID").getInt(), expectedLatestId);
+}
+
+TEST_F(ClusterInfoTest, replan_will_use_latest_id) {
+  VPackBuilder builder;
+  VPackParser parser(builder);  // add json as vpack to the builder
+  std::string test_plan{R"([{"arango":
```

Comment:
I think a much smaller `Plan` could be used here, since the contents of the plan does not actually matter. As far as I know the top level components `Analyzers`, `Databases`, `Collections` and `Views` must exist (whatever `replanAgency` needs), but the gory details do not matter.